gdrive_sync/management/commands/create_missing_website_video_folders.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from gdrive_sync import api
+from websites.models import Website
+
+
+class Command(BaseCommand):
+    """Creates a gdrive folder for video uploads for websites that don't already have one"""
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        if settings.DRIVE_SHARED_ID and settings.DRIVE_SERVICE_ACCOUNT_CREDS:
+            websites = Website.objects.filter(owner_id__isnull=False)
+
+            self.stdout.write(
+                self.style.WARNING(
+                    f"Creating gdrive folders for {websites.count()} website objects if they do not already exist..."
+                )
+            )
+
+            for website in websites:
+                api.create_gdrive_folder_if_not_exists(website.short_id, website.name)
+
+        else:
+            self.stdout.write(self.style.WARNING(f"Google drive credentials not set."))
```

Comment:
Minor thing, it might be helpful to add a stdout statement when processing is complete like in [backpopulate_pipelines](https://github.com/mitodl/ocw-studio/blob/a5df2b7f0870eec4ff3f12247b92a82483f7be6c/content_sync/management/commands/backpopulate_pipelines.py#L64-L70)